WORKSPACE

```diff
@@ -5,6 +5,10 @@ load("//format:repositories.bzl", "rules_format_dependencies")
 # Fetch dependencies which users need as well
 rules_format_dependencies()
 
+load("//format:dependencies.bzl", "rules_format_setup")
```

Comment:
I guess the names no longer make sense, because you now call `rules_format_setup` from `dependencies.bzl` and `rules_format_dependencies` from `repositories.bzl`